src/main/java/com/example/music/backend/user/repository/UserRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.example.music.backend.user.repository;
+
+import com.example.music.backend.user.domain.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {
+
+    Optional<User> findByEmail(String email);
+
+    User findByUserName(String username);
```

Comment:
Why here is " User" but in previous line "Optional<User>"?

Reply:
removed the method as it is not needed